### DISCUSSION TITLE
Problem in generate .p12 certificate

### DISCUSSION THREAD

**Discussion by husin-sajjadi** [SoftEtherVPN/SoftEtherVPN#1846] [General]
Hi!

I've connect to server via softether vpn manager with SSTP protocol.
As you know, SSTP needs a certificate to validation. So I've create a .p12 certificate via Encryption and Network setting in The Softether VPN Server management tool, and set a password (Private Key protection) for this certificate

I installed the .p12 certificate on windows and It installed as well and connected to SSTP server.

But, When I want to install it on Android or IOS devices, OS says that password is incorrect!
I tested these Encryption Algorithm name:
AES256-GCM-SHA384, AES256-SHA256, AES128-GCM-SHA256, AES128-SHA256

Should I change it to another encryption for Android or IOS?

**Comment by shakibamoshiri** [comment]
First, what you have faced is not an issue, it is question and better to be asked here 
https://www.vpnusers.com/viewforum.php?f=7

Second, SSTP needs a valid cert to connect e.g R3 (Let Encrypt) and be loaded to the server. if you do not want to go to the trouble of installing it on each device.

**Reply by CoolDixon** [reply]
I have the same ISSUE than Husin, if you generate a certificate with SoftEther for a given user, when you try to import it in the client machine it says wrong import password, even when I saved it without password or a very basic one such as 'abc'

So it is impossible to get certificate authentication working
